pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -282,6 +282,16 @@ func (a *Operator) ensureTenantRBAC(operatorNamespace, targetNamespace string, c
 				return err
 			}
 		}
+		for _, ns := range pruneNamespaces {
```

Comment:
These RBAC changes and the ones below seem to be unnecessary. Why would that be?